src/coreclr/interpreter/inc/interpretershared.h

```diff
@@ -22,14 +22,22 @@ struct InterpHelperData {
     uint32_t accessType : 3;
 };
 
+#ifndef INTERPRETER_COMPILER_INTERNAL
+class MethodDesc;
+#endif
+
 struct CallStubHeader;
 
 struct InterpMethod
 {
 #if DEBUG
     InterpMethod *self;
 #endif
+#ifdef INTERPRETER_COMPILER_INTERNAL
     CORINFO_METHOD_HANDLE methodHnd;
+#else
+    DPTR(MethodDesc) methodDesc;
```

Comment:
This seems like it won't build since we're undefining methodHnd in favor of methodDesc?